src/catalog/services/df_service.py

```diff
@@ -87,3 +87,19 @@ def dataset_object_by_name(self, database_name, dataset_name,
         """
         return self.model.query.filter(
             self.model._name == dataset_name).one()
+        
```

Comment:
Add test case for this implementation

Reply:
def (test_dataset_object_by_name_queries_with_name_returns_model_object) this testcasse is already present in the test_dataset_service.py